toolchain/check/import_cpp.cpp

```diff
@@ -529,30 +529,30 @@ static auto MapBuiltinType(Context& context, const clang::BuiltinType& type)
 static auto MapRecordType(Context& context, SemIR::LocId loc_id,
                           const clang::RecordType& type) -> TypeExpr {
   auto* record_decl = clang::dyn_cast<clang::CXXRecordDecl>(type.getDecl());
-  if (record_decl && record_decl->isStruct()) {
+  if (record_decl && (record_decl->isStruct() || record_decl->isClass())) {
```

Comment:
```suggestion
  if (record_decl && !record_decl->isUnion()) {
```
... to make it a bit clearer which case we're not yet handling.

Reply:
SGTM, since this is already merged, sent out #5715 for that.